stories/examples/RangeSlider.js

```diff
@@ -1,23 +1,18 @@
-import React from 'react'
-import { RangeSlider } from 'react-ui-core/src'
-import { RangeSliderTheme } from '../theme'
-import { action } from '@storybook/addon-actions'
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { RangeSlider } from "react-ui-core/src";
+import { RangeSliderTheme } from "../theme";
+import { action } from "@storybook/addon-actions";
 
 const wrapper = (props = {}) => {
-  const {
-    minValue,
-    maxValue,
-    theme,
-    value,
-  } = props
+  const { minValue, maxValue, theme, value } = props;
```

Comment:
not sure why this is being  changed. We typically put things on multiple rows when there are more than a few items, as it was originally. Also, you can remove the semicolon.